nms/app/packages/magmalte/app/views/equipment/__tests__/EnodebConfigTest.js

```diff
@@ -436,6 +547,87 @@ describe('<AddEditEnodeButton />', () => {
           tac: 1,
           transmit_enabled: true,
         },
+        enodeb_config: {
+          config_type: 'MANAGED',
+          unmanaged_config: undefined,
+          managed_config: {
+            bandwidth_mhz: 20,
+            cell_id: 1,
+            device_class: 'Baicells ID TDD/FDD',
+            earfcndl: 40000,
+            pci: 36,
+            special_subframe_pattern: 7,
+            subframe_assignment: 2,
+            tac: 1,
+            transmit_enabled: true,
+          },
+        },
+        description: 'test enodeb description',
+        name: 'testEnodeb0',
+        serial: 'testEnodebSerial0',
+      },
+      enodebSerial: 'testEnodebSerial0',
+      networkId: 'mynetwork',
+    });
+  });
+
+  it('Verify Enode Edit unmanaged eNodeB', async () => {
+    const {getByTestId, getByText, queryByTestId} = render(<DetailWrapper />);
+    await wait();
```

Comment:
I think we'll need to update these to waitFor, probably in a later PR

Reply:
Yes. I think await wait() is used across. Will clean this up in one NMS test cleanup.